src/page-response-impact/ResponseImpactDashboard.tsx

```diff
@@ -152,6 +161,22 @@ const ResponseImpactDashboard: React.FC = () => {
     fetchFacilities();
   }, [scenarioState?.data?.id]);
 
+  // calculate data for cards
```

Comment:
This component is starting to get pretty big. I don't think its necessary to do it in this PR but we should probably start thinking about breaking it up.

Reply:
hah for sure. You should have seen it before I moved a couple functions/types to the Utils. 

Depending on the other components' needs, I might be able to move all of this down to the card container component.